packages/xdl/src/start/ExpoUpdatesManifestHandler.ts

```diff
@@ -119,6 +119,10 @@ export async function getManifestResponseAsync({
     platform
   );
 
+  if (!runtimeVersion) {
```

Comment:
cc @jkhales - if `Updates.getRuntimeVersion(..)` returns null then we can handle that here. i believe that this particular function is where not having a runtime version available warrants an exception, because we can't build a valid manifest (according to the type) without a runtimeVersion string

Reply:
maybe we can get rid of `getRuntimeVersionNullable` and have `getRuntimeVersion` behave as it does in this pr? i think the caller should handle whether having null for runtime version is an exception or not

Reply:
That sounds great to me!